src/Stratis.Bitcoin.Features.Consensus/CoinViews/Coindb/LeveldbCoindb.cs

```diff
@@ -186,6 +221,20 @@ public void SaveChanges(IList<UnspentOutput> unspentOutputs, HashHeightPair oldB
             this.performanceCounter.AddInsertedEntities(insertedEntities);
         }
 
+        /// <inheritdoc />
+        public int GetMinRewindHeight()
+        {
+            // Trim all rewind blocks prior to last checkpoint.
+            using (var iterator = this.leveldb.CreateIterator())
+            {
+                iterator.Seek(new byte[] { rewindTable });
+                if (!iterator.IsValid())
+                    return -1;
+
+                return BitConverter.ToInt32(iterator.Key().Reverse().ToArray());
```

Comment:
As you are now storing it in reverse, shouldn't this just return the key as is?

Reply:
BitConverter will still need it in the un-reversed order.